src/components/Navigation/navigation.scss

```diff
@@ -0,0 +1,126 @@
+@import 'globals';
+
+$menu-breakpoint: map-get($breakpoints, 'navigation-menu');
+
+:export {
+  menuBreakpoint: $menu-breakpoint;
+}
+
+/**
+ * 1. Extra 1% so that it goes under the nav's grey border.
+ */
+.overlay {
+  position: absolute;
+  top: 101%; /* 1 */
```

Comment:
seems a bit hacky? can we calc something here based on exactly the border width?